src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -295,6 +318,32 @@ private void UpdateDescriptionVisibility(bool initialization)
 			}
 		}
 
+		public static DependencyProperty IsEditableProperty { get; } =
```

Comment:
As `IsEditable` is not implemented yet, please move these back to generated file

Reply:
Done.